tools/test.py

```diff
@@ -152,8 +152,7 @@
         else:
             # Build all tests
             if not options.build_dir:
-                print "[ERROR] You must specify a build path"
-                sys.exit(1)
+                args_error(parser, "argument --build is required")
```

Comment:
This used return code  1 on exit, but `args_error` always uses 2. Not sure if this is actually a problem or not, but it might break some code that checks for the return code.